enhancements/ota/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,165 @@
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+### Cincinnati Git(Hub) scraper deprecation
+We need to decide whether we want to maintain the Git(Hub) scraper plugin in the future or drop it altogether.
+
+
+### Content trust
+Trust in the container image registry and in the container image content is possibly an issue if we an approach where Cincinnati uses a mutable tag, e.g. `latest` tag to receive the most recent content from the container registry.
+Assuming we're using Quay as an image registry, [which does not support Docker Content Trust](https://support.coreos.com/hc/en-us/articles/115000289114-Docker-Content-Trust-support).
+If we decide that to ensure the image integrity, which means do we have which do not further complicate the workflow?
+
+### Product name for the CPaaS integration
+Which product name should this image be released under?
+Related to this is that it's currently unclear if the Cincinnati image itself will be published under its own product name or now.
+Further note: the repository for the graph-data images will not be semantically versioned.
+
+### Repository location and name
+Under which well-known container image repository will we publish the images?
+
+
+### Continuous build and release
+Can CPaaS handle continuous build and release from master?
+
+## Summary
+The [cincinnati-graph-data GitHub repository][cincinnati-graph-data] will be continuously packaged and released in form of a container image by the means of integration with CPaaS.
+This image will be available for public consumption, and will be used by the publicly available Cincinnati instances operated by Red Hat.
+The image can be transferred and served to Cincinnati instances operated by any party, including but not limited to network-restricted environments.
+The source of the container image, namely the cincinnati-graph-data, will remain publicly available and its workflows unaffected by this enhancement.
+
+## Motivation
+
+### Goals
+- The container image will become the primary way of consuming and distributing the content of the cincinnati-graph-data repository.
+- As long as the primary release metadata will be packaged, distributed and consumed in form of container images as well, this streamlines all transport mechanisms required for data consumption by Cincinnati, as well as mirroring and serving its consumed content.
+- Every change to the master branch of the cincinnati-graph-data results in a new container image containing the unmodified working tree files of the new revision.
+- The image will be pushed to a publicly available container image registry und a well-known name.
+
+### Non-Goals
+- Changes to the GitHub repository workflow or data format and make progress.
+- Implementation details about the streamlining of the mirroring process for primary and secondary release metadata.
+
+## Proposal
+
+### User Stories [optional]
```

Comment:
nit: I think `[optional]` is a template-side marker that the section can be dropped when writing an enhancement.  I don't think enhancements that keep the section should still drop the `[optional]` marker, although existing enhancements in this repo are not consistent about that.

Reply:
> I don't think enhancements that keep the section should still drop the `[optional]` marker

Just to clarify. You mean you *do* think so, right?

Reply:
> Just to clarify. You mean you do think so, right?

Hah, sorry.  Yes, I think you are using the section, and I think that you should drop the `[optional]` marker.